test/models/concerns/audit_extensions_test.rb

```diff
@@ -46,4 +46,13 @@ def setup
     FactoryGirl.create(:ec2_cr, :with_auditing)
     refute_empty Audit.search_for("type = compute_resource")
   end
+
+  test "audit change field can be greater then 65K bytes" do
+    prov_template = templates(:mystring)
+    prov_template.template = "0" * 66000
+    as_admin do
+      assert prov_template.save!
+    end
+    assert_equal 66000, prov_template.template.bytesize
```

Comment:
shouldn't this check the audited changes?
Also, afaict it doesn't look like the template field itself supports medium text, in fact the only column i found using mediumtext is the log column in the reports table.